### PR TITLE
ES Linting lib

### PR DESCRIPTION
**Pull request by xjunior**
- Introduce @powerhome/eslint-config library
- Initial github workflow

### CODE REVIEW

File: .github/workflows/eslint-config.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          registry-url: ${{ env.REGISTRY_URL }}
+      - run: npm ci
+        working-directory: ${{ env.PROJECT_DIR }}
+      - run: npm publish
+        working-directory: ${{ env.PROJECT_DIR }}
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (benlangfeld) [inline .github/workflows/eslint-config.yml:49]: I configured this as a token on my account. Should be good to go.

File: eslint-config/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  ],
+  ...
+}
+```
+
+That's it! You can override the standard configuration after that.
+
+## Flow type apps
+
+For flow-based apps, replace `@powerhome` by `@powerhome/eslint-config/flow`.
```
Comment (benlangfeld) [inline eslint-config/README.md:40]:
```suggestion
For flow-based apps, replace `@powerhome` by `@powerhome/eslint-config/flow`. Note that usage of Flow at Power is not recommended, and all projects should migrate to TypeScript; this set of rules is provided only for transitionary purposes.
```

File: eslint-config/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,2096 @@
+{
```
Comment (benlangfeld) [inline eslint-config/package-lock.json:1]: Why do you have this *and* a yarn.lock?

File: .github/workflows/eslint-config.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v3
+        with: 
+          node-version: ${{ env.NODE_VERSION }}
+      - run: npm ci
+        working-directory: ${{ env.PROJECT_DIR }}
+      - name: Run the build script
+        run: npm run lint
```
Comment (benlangfeld) [inline .github/workflows/eslint-config.yml:24]: Looks like this didn't run.

**Review by benlangfeld — CHANGES_REQUESTED** [review]

**Review by wadewinningham — APPROVED** [review]

**Pull request merged**